HW2_Arnaut/calculator.py

```diff
@@ -0,0 +1,33 @@
+
+from decimal import Decimal
+
+def minus(a, b):
+    return a - b
+
+def plus(a, b):
+    return a + b
+
+def product(a, b):
+    return a * b
+
+def divide(a, b):
+    if b!=0:
```

Comment:
Не забывайте про отступы!
```suggestion
    if b != 0:
```